keyboards/keychron/q5/ansi/keymaps/vial/config.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200  // Sets the default brightness value, if none has been set
+
+//Vial Keyboard UID
+//#define VIAL_KEYBOARD_UID {0x5C, 0x0E, 0xC0, 0x71, 0x7A, 0x35, 0x35, 0xAD}
+
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 2 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 13 }
+
+#define VIAL_KEYBOARD_UID {0xAA, 0xA6, 0x8C, 0x02, 0x73, 0x1D, 0x30, 0x51}
+ 
+// Number of layers
+// #define DYNAMIC_KEYMAP_LAYER_COUNT 4
+#define LAYER_STATE_16BIT
+ 
+// Caps word
+#define BOTH_SHIFTS_TURNS_ON_CAPS_WORD
+#define DOUBLE_TAP_SHIFT_TURNS_ON_CAPS_WORD
+#define CAPS_WORD_IDLE_TIMEOUT 5000
```

Comment:
User-custom code shouldn't be added in the default `vial` keymaps. Should be stock.  
```suggestion
#define VIAL_KEYBOARD_UID {0xAA, 0xA6, 0x8C, 0x02, 0x73, 0x1D, 0x30, 0x51}

#define VIAL_UNLOCK_COMBO_ROWS { 0, 2 }
#define VIAL_UNLOCK_COMBO_COLS { 0, 13 }
```